common/constants/custom_panels.ts

```diff
@@ -6,7 +6,7 @@
 export const CUSTOM_PANELS_API_PREFIX = '/api/observability/operational_panels';
 export const CUSTOM_PANELS_DOCUMENTATION_URL =
   'https://opensearch.org/docs/latest/observability-plugin/operational-panels/';
-export const CREATE_PANEL_MESSAGE = 'Enter a name to describe the purpose of this custom panel.';
+export const CREATE_PANEL_MESSAGE = 'Enter a name to describe the purpose of this observability dashboard.';
```

Comment:
I feel we should properly capitalize the object names in messaging : "Observability Dashboard"

Reply:
Done